operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -416,4 +485,272 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
                 .value(requestBaseURI + "/" + uriPath)
                 .build();
     }
+    
+    private void addIncludesSearchParameters(String compartmentType, List<String> typeParameters, MultivaluedMap<String, String> searchParameters) {
```

Comment:
I find the name `compartmentType` a bit confusing/ambiguous.
A.  The type of the compartment being searched (Patient)
B.  The resource type to search within the Patient compartment

I know we mean `B` here, but I think we should either update the variable name (here and elsewhere) to be more clear, or add some javadoc (despite it being a private method) to clarify.

Reply:
for an alternative name, maybe something like "searchResourceType" or "matchResourceType" (to distinguish from the "includeResourceType")

Reply:
OK, changed to compartmentMemberType